src/content/1.7/scale/optimizations.md

```diff
@@ -46,38 +66,68 @@ post_max_size = 22M
 ; Increase this value if you work with products with a lot of combinations
 max_input_vars = 20000
 allow_url_fopen = on
+```
+### 2) PHP & File system
+
+It is well known that PHP does not manage file systems very well.
+
+That's why there are plenty of useful tuning options to avoid constent file system's accesses.
+
+#### realpath_cache
+
+At each file access, by default, PHP will first check that the file is still there, causing plenty of `lstat` system calls. Potentially thousands.
+
+To avoid this, PHP allows to cache the files in a cache, then avoiding those system calls:
 
+```
+[PHP]
+realpath_cache_size = 4096K
+realpath_cache_ttl = 600
+```
+
+Keep in mind that this configuration is NOT compatible with other parameters, such as `open_basedir` and `safe_mode` directives.
+
+Finally, if you're using a NAS or any other network storage solution to store your files (in case of horizontal scaling, for example), it is more than recommeneded to enable this parameter.
```

Comment:
```suggestion
Finally, if you're using a NAS or any other network storage solution to store your files (in case of horizontal scaling, for example), it is **highly recommended** to enable this setting.
```